webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -16,7 +17,7 @@ class BreadcrumbBar extends React.Component {
       resourceSwitcherItems,
     } = this.props;
     const isUrlFormatValid = resourceSwitcherItems.length
-      ? resourceSwitcherItems[0].url ===
+      ? resourceSwitcherItems[0].href ===
```

Comment:
this is to fix a bug where the resources didn't update when moving from edit host to show host (https://github.com/theforeman/foreman/pull/6382)
Before 6382 - the resources was loaded one for edit hosts
after 6382 - the resources were loaded on every open as `resourceSwitcherItems[0].url` is undefined 
after changing to `href` - the resources are loaded only when the breadcrumbs are on a different page